src/translations/nb_NO.json

```diff
@@ -123,7 +123,7 @@
   "SIDEBAR_REQUEST_CANCELLATION_GENERIC_INSURANCE": "Kanseller min gamle forsikring",
   "RETRY": "Prøv igjen",
   "CONNECT_PAYMENT_DIRECT_ERROR_TAKING_TOO_LONG": "Det tok for lang tid å laste siden, vennligst prøv igjen.",
-  "CONNECT_PAYMENT_DIRECT_ERROR_INVALID_TOKEN": "Koblingen du skrev inn er en ugyldig lenke.\\n",
+  "CONNECT_PAYMENT_DIRECT_ERROR_INVALID_TOKEN": "Koblingen du skrev inn er en ugyldig lenke.\n",
```

Comment:
Not related to the PR itself, but just curious why there's a line break at the end of the sentence?

Reply:
I have no idea 😄 But apparently it's been there for a while (only it's been written out as `\n` I suppose)